README.md

```diff
@@ -49,3 +49,40 @@ real openshift-ansible image to use in the ANSIBLE_IMAGE variable.
 	* `USE_REAL_AWS=1 contrib/examples/create-cluster.sh`
 	* `ANSIBLE_IMAGE=openshift/origin-ansible:latest contrib/examples/create-cluster.sh`
 
+## Developing With OpenShift Ansible
+
+You can build your own custom version of OpenShift Ansible when working on playbooks and roles from the openshift-ansible repo:
```

Comment:
Could we link to the openshift-ansible repo here?

Reply:
Oh and could you fix the bulleted list at the bottom of the README while you're here :bowtie:

Reply:
Added the link, could you clarify what's wrong with a bulleted list, I'm not seeing anything.

Reply:
Lines 49/50 right above your changes are indented with tabs or something so they're just being shown inline rather than as a list.